frontend/src/firebase.ts

```diff
@@ -0,0 +1,22 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getFirestore, Firestore } from "firebase/firestore";
+
+// TODO: Add SDKs for Firebase products that you want to use
```

Comment:
Problem with TODO is that it doesn't help you actually remember what to do when you are planning your sprint. Better just put something in jira